app/scripts/modules/kubernetes/serverGroup/details/details.html

```diff
@@ -74,6 +74,8 @@ <h4 class="text-center" ng-if="serverGroup.isDisabled">[SERVER GROUP IS DISABLED
         <dd ng-show="serverGroup.revision">{{serverGroup.revision}}</dd>
         <dt>Kube UI</dt>
         <dd><a href={{ctrl.uiLink()}} target="_blank">{{serverGroup.name}}</a></dd>
+        <dt>TERM Period</dt>
```

Comment:
Not sure what you mean by change the layout's width - but you're welcome to break from the existing `<dt>...</dt><dd>...</dd>` pattern to make the label fit. The Kubernetes load balancer details panel does this.